.github/workflows/gke.yaml

```diff
@@ -57,6 +57,8 @@ jobs:
       operator_nightly_chart: ${{ inputs.operator_nightly_chart == true || (github.event_name == 'schedule' && true) }}
       run_p0_provisioning_tests: ${{ contains(inputs.tests_to_run, 'p0_provisioning') || (github.event_name == 'schedule' && true) }}
       run_p0_importing_tests: ${{ contains(inputs.tests_to_run, 'p0_importing') || (github.event_name == 'schedule' && true) }}
+      run_p1_provisioning_tests: ${{ contains(inputs.tests_to_run, 'p1_provisioning') || (github.event_name == 'schedule' && true) }}
```

Comment:
```suggestion
      run_p1_provisioning_tests: ${{ contains(inputs.tests_to_run, 'p1_provisioning') || (github.event_name == 'schedule' && false) }}
```

Reply:
Good catch! I understand your earlier query about running it daily now :sweat_smile: